tests/expressions.cpp

```diff
@@ -219,21 +219,19 @@ struct ExpressionsFixture : public TiledRangeFixture {
 typedef ExpressionsFixture<TiledArray::Tensor<int>, TA::SparsePolicy>
     EF_TAspTensorI;
 
-typedef ExpressionsFixture<TiledArray::Tensor<std::complex<int>>,
-                           TA::DensePolicy>
-    EF_TATensorCI;
+typedef ExpressionsFixture<TiledArray::Tensor<int>,TA::DensePolicy>
+    EF_TATensorI;
 
 #ifdef TILEDARRAY_HAS_BTAS
 typedef ExpressionsFixture<
     TA::Tile<btas::Tensor<int, TiledArray::Range, btas::varray<int>>>,
     TA::DensePolicy>
     EF_TABtasI;
 
-typedef boost::mpl::vector<EF_TAspTensorI, EF_TATensorCI,
-                           EF_TABtasI>
+typedef boost::mpl::vector<EF_TAspTensorI, EF_TATensorI,EF_TABtasI>
```

Comment:
to avoid memory issues might be sufficient to split into multiple .cpp files, which each testing only 1 tensor type